installInstructions/install3rdPartyPackages.rst

```diff
@@ -50,6 +50,12 @@ If they're not available already, common prerequisites can be installed directly
    sudo apt-get install python-dev gfortran valgrind cmake libblas-dev liblapack-dev
```

Comment:
Instead of this note, I would be inclined to just append the `build-essential` package here. 
The `build-essential` package does provide several other packages, such as `gcc` and `make`, and thus this prerequisite line would serve as a more complete setup, in case the user has not already installed e.g., `gcc`.

Reply:
That makes a lot of sense! The full list of packages in the bundle for Ubuntu 20.04 is [here](https://packages.ubuntu.com/focal/build-essential). Shall I just append `build-essentials` or do we want to also take out some packages, e.g. gfortran and cmake? Not sure about how much `build-essentials` overlaps with these.

Reply:
From what I can tell it does not overlap with anything we currently have, such as `cmake` or `gfortran`, so I think it's fine to just append to the list.

Reply:
sounds good to me, I am editing this to include `build-essentials` among the packages